python/pyarrow/tests/parquet/test_encryption.py

```diff
@@ -38,7 +38,9 @@
 
 # Marks all of the tests in this module
 # Ignore these with pytest ... -m 'not parquet_encryption'
+# Ignore these with pytest ... -m 'not parquet'
 pytestmark = pytest.mark.parquet_encryption
+pytestmark = pytest.mark.parquet
```

Comment:
Ah, we can't assign two marks this way (it is redefining the same variable), it should be something like `pytestmark = [..., ...]` I think